packages/api/src/functions/graphql.ts

```diff
@@ -1,17 +1,23 @@
 import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
 import type { Config, CreateHandlerOptions } from 'apollo-server-lambda'
 import type { Context, ContextFunction } from 'apollo-server-core'
-import type { GlobalContext } from 'src/globalContext'
-import type { AuthContextPayload } from 'src/auth'
+import type { GlobalContext } from '../globalContext'
+import type { AuthContextPayload } from '../auth'
+import type { APIGatewayProxyCallback } from 'aws-lambda'
 import { ApolloServer } from 'apollo-server-lambda'
-import { getAuthenticationContext } from 'src/auth'
-import { setContext } from 'src/globalContext'
+import { getAuthenticationContext } from '../auth'
+import { setContext } from '../globalContext'
```

Comment:
Curious, why is "../" preferred over "src"?

Reply:
See #1381 it'll be addressed there, if fixable.